modules/codemod/lib/v9/compoundToast.ts

```diff
@@ -0,0 +1,227 @@
+import {API, ASTPath, FileInfo, ImportDeclaration, JSXElement, Options} from 'jscodeshift';
+import {getImportRenameMap} from '../v7/utils/getImportRenameMap';
+import {hasImportSpecifiers} from '../v6/utils';
```

Comment:
I've seen other files do this, is this ok?

Reply:
Yes, I'd like to refactor utils in a separate PR when possible